hack/perf-ibmcloud-openshift.sh

```diff
@@ -314,6 +333,13 @@ install_demo_apps() {
 }
 
 install_workloads() {
+  SCC_SERVICE_ACCOUNTS=""
+  for (( c=0; c<WORKLOAD_NAMESPACES; c++ ))
+  do
+    SCC_SERVICE_ACCOUNTS="${SCC_SERVICE_ACCOUNTS}- \"system:serviceaccount:depth-$c:default\"
+"
```

Comment:
this looks weird. Is this syntax what you meant? (quote all by its lonesome on a separate line?)